src/ucp/proto/proto_common.c

```diff
@@ -497,7 +525,12 @@ ucp_proto_common_find_lanes(const ucp_proto_init_params_t *params,
     }
 
     lane_map = UCS_MASK(ep_config_key->num_lanes) & ~exclude_map;
-    ucs_for_each_bit(lane, lane_map) {
+    lane     = 0;
+    for (i = ucp_proto_common_lanes_iter(ep_config_key, lane_map, lane_type,
```

Comment:
maybe add a variable instead to make it more readable?

Reply:
After f2f discussion we decided to try another approach - sort lines in proto_multi taking into account not only BW, but also paths, latency etc. So this code is not relevant anymore

Reply:
ok, still we decided to go on with this quickfix, before we implement the better solution
Sorry, which variable do yo mean?